src/import-data/import-utils.ts

```diff
@@ -1,8 +1,27 @@
-import { canWrite, WorkspaceWrapper } from 'src/libs/workspace-utils';
+import { Snapshot } from 'src/libs/ajax/DataRepo';
+import { canWrite, CloudProvider, getCloudProviderFromWorkspace, WorkspaceWrapper } from 'src/libs/workspace-utils';
 
+import { ImportRequest } from './import-types';
 import { isProtectedWorkspace } from './protected-data-utils';
 
+export const getCloudPlatformRequiredForImport = (importRequest: ImportRequest): CloudProvider | undefined => {
+  switch (importRequest.type) {
+    case 'tdr-snapshot-export':
+    case 'tdr-snapshot-reference':
+      const tdrCloudPlatformToCloudProvider: Record<Snapshot['cloudPlatform'], CloudProvider> = {
+        azure: 'AZURE',
```

Comment:
The backend is going to enforce that you're not allowed to import AZURE snapshots by ref _at all_ see some conversation and relevant code here: https://github.com/broadinstitute/rawls/pull/2556#discussion_r1342919480

Would it make sense to replicate that constraint here too until we actively build support for it?

Reply:
Maybe snapshot-by-reference should be removed for all platforms? I wonder if it's being used at all right now, except by some e2e tests.

Reply:
snapshot-by-reference is still in active use.

Reply:
> The backend is going to enforce that you're not allowed to import AZURE snapshots by ref at all ...
> Would it make sense to replicate that constraint here too until we actively build support for it?

😞 I may have to get this one in a separate PR. I think it needs to be handled a bit differently since there's _no_ way to do an import by reference of an Azure snapshot.

Reply:
#4332 blocks imports by reference of Azure snapshots.

Reply:
> snapshot-by-reference is still in active use.

I know it's being used by API users, but didn't know if it was being used by UI users.

> That said... maybe we could remove the import type for multiple snapshots by reference from the catalog?

That would be OK with me, it's not being used right now and it can be added back later when/if the need arises.

Reply:
> That would be OK with me, it's not being used right now and it can be added back later when/if the need arises.

Opened #4353 to remove it.